Media/Model/MediaInterface.php

```diff
@@ -143,6 +143,22 @@ public function removeUsageReference($reference);
     public function getUsageReference();
 
     /**
+     * @param string $formatName
+     * @param string $alternativeName
+     */
+    public function addAlternative($formatName, $alternativeName);
+
+    /**
+     * @param string $formatName
+     */
+    public function removeAlternative($formatName);
+
+    /**
+     * @return array
+     */
+    public function getAlternatives();
```

Comment:
Maybe you can add a getAlternative($formatName), it seems usefull for an integrator, but I don't know if we will trully need it